unity-renderer/Assets/Scripts/MainScripts/DCL/WorldRuntime/KernelCommunication/WebSocketCommunication/DCLWebSocketService.cs

```diff
@@ -32,6 +36,23 @@ private void SendMessageToWeb(string type, string message)
 #endif
     }
 
+    private void SendBinaryMessageToWeb(string sceneId, byte[] data)
```

Comment:
Can we change this to `SendBinaryMessageToKernel`?

I know that we use `Web` like `WebInterface` and more in a lot of places. But I think `Kernel` is a better naming. Do you agree?